backend/development.ini.sample

```diff
@@ -532,7 +532,7 @@ website.usage_conditions =
 # Possible events are in the form ENTITY_TYPE.EVENT_TYPE or ENTITY_TYPE.EVENT_TYPE.CONTENT_TYPE where:
 # ENTITY_TYPE  = user_call | user | workspace | content | mention | workspace_member
 # EVENT_TYPE   = created | modified | deleted | undeleted
-# CONTENT_TYPE = html-document | file | thread | folder | comment
+# CONTENT_TYPE = html-document | file | thread | folder | comment | logbook
```

Comment:
I know this is not PR for that but do you know if kanban nee to be in this list ? (or if it is normal to not seen this app)

Reply:
#6399